.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Environment variables should not be left blank as it may lead to misconfiguration. Ensure that the values for GITHUB_TOKEN and OPENAI_API_KEY are securely set and validated before use, to prevent potential bugs at runtime.